src/locales/en.json

```diff
@@ -478,27 +478,51 @@
             }
         },
         "UpdatePanel": {
+            "Abort": "Abort",
+            "AreYouSure": "Are you sure?",
             "CheckForUpdates": "Check for updates",
+            "Close": "Close",
+            "CommitHistory": "Commit History",
             "Commits": "Commits",
             "CommitsAvailable": "no commits available | {count} commit available | {count} commits available",
             "CommitsOnDate": "Commits on {date}",
             "CommittedDaysAgo": "committed {days} days ago",
             "CommittedHoursAgo": "committed {hours} hours ago",
             "CommittedOnDate": "committed on {date}",
             "CommittedYesterday": "committed yesterday",
+            "ConfigChanges": "Config Changes",
+            "CountPackagesCanBeUpgraded": "{count} packages can be upgraded",
             "Detached": "detached",
             "Dirty": "dirty",
-            "ERROR": "ERROR",
+            "GenericUpdateQuestion": "Check the commit history (if available) and the GitHub page for this project to see if any manual adjustments are necessary for this update.",
+            "GitHubPage": "GitHub Page",
+            "HardRecovery": "Hard Recovery",
+            "InitUpdateManager": "The Update Manager has not been initialized yet. This is normal when you start the system for the first time. Please click on the refresh button to initialize all components.",
             "Invalid": "invalid",
+            "IUnderstandTheRisks": "I understand the risks",
+            "LinkToGithub": "Link to GitHub",
+            "KlipperUpdateQuestionFirmware": "This will update the host Klipper software. Control boards that you have flashed the Klipper firmware onto may need to be recompiled and reflashed before you can continue using your machine.",
+            "KlipperUpdateQuestionConfig": "This update may also contain changes to config parameters that would need to be modified in your printer.cfg file, see the change log for details.",
+            "MoonrakerUpdateQuestion": "This will update the Moonraker API. Changes to your moonraker.conf file may be required to continue using your machine.",
```

Comment:
```suggestion
            "MoonrakerUpdateQuestion": "This will update the Moonraker API. Changes to the moonraker.conf file may be required to continue using the machine.",
```